packages/keystone/file/utils.js

```diff
@@ -11,6 +11,8 @@ const FORCE_ENCODING_CHANGE = {
     'KOI8-R': 'WINDOWS-1251', // TODO: Find why KOI8 detected on WINDOWS-1251
     'ISO-8859-2': 'WINDOWS-1251',
 }
+const ROWS_COUNT_LIMIT = 100_000
+const ROWS_COUNT_LIMIT_EXCEEDED = 'Rows count limit exceeded'
```

Comment:
Please move these constants to a separate file. Otherwise, we have to load all packages from utils.js when we need only two constants.